src/frontend/src/components/lifestyle_data.js

```diff
@@ -61,75 +80,164 @@ export default class LifestyleData extends React.Component {
+                        <MenuItem id="sleepMenuItem23" value={23}>23:00</MenuItem>
+                        <MenuItem id="sleepMenuItem0" value={0}>00:00</MenuItem>
+                        <MenuItem id="sleepMenuItem1" value={1}>1:00</MenuItem>
+                        <MenuItem id="sleepMenuItem2" value={2}>2:00</MenuItem>
+                        </Select>
+                    </FormControl>
+                    <bs.Col lg={4}></bs.Col>
+                </bs.Row>
+                <br/>
+                <bs.Row>
+                    <bs.Col lg={4}></bs.Col>
+                    <Button id="controlDropdownWake" className={useStyles.button} onClick={()=>{
+                        this.setState({
+                            wakeUpDropDownOpen : true
+                        })
+                    }}>
+                        Waking Up Time
+                    </Button>
+                    <FormControl className={useStyles.formControl}>
+                        
+                        <Select
+                        labelId="demo-controlled-open-select-label"
+                        id="selectWake"
+                        open={this.state.wakeUpDropDownOpen}
+                        onClose={()=>{
+                            this.setState({
+                                wakeUpDropDownOpen : false
+                            })
                         }}
-                        inputProps={{
-                            step: 300,
+                        onOpen={()=>{ 
+                            this.setState({
+                                wakeUpDropDownOpen : true
+                            })
                         }}
-                    />
-                </form>
+                        value={this.state.wakeup_time}
+                        onChange={(e)=> this.handleChange("wakeup_time",e.target.value)}
+                        >
+                        <MenuItem id="wakeMenuItem5" value={5}>5:00</MenuItem>
+                        <MenuItem id="wakeMenuItem6" value={6}>6:00</MenuItem>
+                        <MenuItem id="wakeMenuItem7" value={7}>7:00</MenuItem>
+                        <MenuItem id="wakeMenuItem8" value={8}>8:00</MenuItem>
+                        <MenuItem id="wakeMenuItem9" value={9}>9:00</MenuItem>
+                        <MenuItem id="wakeMenuItem10" value={10}>10:00</MenuItem>
+                        <MenuItem id="wakeMenuItem11" value={11}>11:00</MenuItem>
+                        </Select>
+                    </FormControl>
+                    <bs.Col lg={4}></bs.Col>
+                </bs.Row>
+                <br/>
+                <bs.Row>
+                    <bs.Col lg={4}></bs.Col>
+                        <bs.Col>
+                            <Field >Diet</Field>
+                            <input id="lifestyle_diet" name="diet" placeholder={this.state.diet}
+                             onChange={(e)=>this.handleChange("diet",e.target.value)}/>
```

Comment:
Codacy has a fix for the issue: [Missing space before =>.](https://app.codacy.com/gh/MarkHershey/SUTDHousingPortal/pullRequest?prid=7298734)
```suggestion
                             onChange={(e) =>this.handleChange("diet",e.target.value)}/>
```